resources/servo.css

```diff
@@ -269,6 +269,9 @@ svg > * {
   contain: unset;
   container: unset;
   scroll-margin: unset;
+
+  /* The grid needs to be block-level, avoid inheriting `display: inline-table` */
```

Comment:
```suggestion
  /* The grid needs to be block-level, so avoid inheriting `display: inline-table`/ */
```

Reply:
I guess at the end it was supposed to be `.` instead of `/`.